.travis-ci.sh

```diff
@@ -9,6 +9,17 @@ COVERITY_SCAN_BUILD_URL="https://scan.coverity.com/scripts/travisci_build_coveri
 
 if [[ $TASK = 'lint' ]]; then
   # run the lint tool only if it is the requested task
+  # first check we've not got any generic NOLINTs
+  # count the number of generic NOLINTs
+  nolints=$(grep -IR NOLINT * | grep -v "NOLINT(" | wc -l)
+  if [[ $nolints -ne 0 ]]; then
+    # print the output for info
+    echo $(grep -IR NOLINT * | grep -v "NOLINT(")
```

Comment:
I can't see anything in the cpplint regex saying it has to be on a line ending though:
https://github.com/google/styleguide/blob/gh-pages/cpplint/cpplint.py#L517

It would seem negative lookahead avoids the extra grep line, this seems to work: